apps/prairielearn/src/tests/groupExam.test.sql

```diff
@@ -1,11 +1,12 @@
--- BLOCK select_group_work_exam_assessment
+-- BLOCK select_group_exam_by_tid
 SELECT
   a.id
 FROM
   assessments AS a
   JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
 WHERE
   a.course_instance_id = 1
+  AND a.tid = $assessment_tid
   AND aset.abbreviation = 'E'
   AND a.group_work is TRUE;
```

Comment:
You could probably remove these two conditionals, as `tid` will be unique within a course instance.

```suggestion
  AND a.tid = $assessment_tid;
```